blueprintapi/tests.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.test import APITestCase
+
+
+class UnauthenticatedAPITestCase(APITestCase):
+    def test_unauthenticated_request_returns_403(self):
+        test_cases = ('/api/projects/', '/api/users/')
```

Comment:
This list certainly isn't exhaustive, but it's really only meant to test that without a token, you get an unauthenticated error. Some endpoints don't have these restrictions though.